docs/howtos/another_kolibri_instance.md

```diff
@@ -0,0 +1,55 @@
+## Running Another Kolibri Instance Alongside the Development Server
+
+This guide will walk you through the process of setting up and running another instance of Kolibri alongside your development server using the `pex` executable.
+
+## Introduction
+
+As Kolibri's features continue to expand into remote content browsing, it's often necessary to test and experiment with another Kolibri instance running alongside your development server. One effective approach is to use the `pex` executable. This workflow is straightforward and can be employed independently of ZeroTier or even internet network access. Documenting these steps will provide a handy reference for both new team members and contributors.
+
+## Prerequisites
+
+Before you begin, make sure you have the following:
+
+- A recent `pex` executable of Kolibri. You can download this from the Build Assets page of a workflow build.
+
+## Steps
+
+- **Locate the .pex executable:**
+
+  - Navigate to the [Kolibri GitHub repository](https://github.com/learningequality/kolibri).
+  - Click on the "Actions" tab at the top of the repository.
+  - Select a workflow build from the list.
+  - Scroll down the workflow build page to find the "Artifacts" section. In this section, you will find the `.pex` file that you need to download. Click on the `.pex` file to initiate the download.

```

Comment:
Given that this is a guide for a developer user, maybe the last phrase about _Click to download..._ is a bit redundant...? 🙂